### PR TITLE
Burger menu and media queries

### PR DESCRIPTION
**Pull request by iObrovac**

### CODE REVIEW

File: src/components/auth/Auth.jsx [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
-
+export default function Auth({ closeMobile, openLogin, openNewAcc }) {
   return (
     <>
       <div>
-        <button className="login" onClick={() => setOpenModal(true)}>
+        <button
+          className="login"
+          onClick={() => {
+            console.log("stiso login");
```
Comment (kiljee) [inline src/components/auth/Auth.jsx]: remove console.log

File: src/components/nav/MobileNavigation.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      size="40px"
+      color="#eaa79e"
+      onClick={() => setOpen(!open)}
+    />
+  );
+
+  return (
+    <nav className="mobile-navigation">
+      {open ? closeIcon : burgerIcon}
+
```
Comment (kiljee) [inline src/components/nav/MobileNavigation.jsx]: remove space line

File: src/components/nav/MobileNavigation.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      onClick={() => setOpen(!open)}
+    />
+  );
+
+  const closeIcon = (
+    <FaWindowClose
+      className="burger"
+      size="40px"
+      color="#eaa79e"
+      onClick={() => setOpen(!open)}
```
Comment (kiljee) [inline src/components/nav/MobileNavigation.jsx]: create function setOpen(!open) and call you have twice same code

File: src/components/nav/Nav.jsx [hunk 4]
```diff
@@ -14,18 +20,17 @@ export default function Nav() {
...
-          Latest Sightings
-        </Link>
-        <Link to="/favorites" className="nav-links">
-          Favorites
-        </Link>
-        <Auth />
-      </div>
+
+      <Navigation
+        openLogin={() => setOpenLogin(true)}
```
Comment (kiljee) [inline src/components/nav/Nav.jsx]: create function use same code twice

File: src/components/nav/Nav.scss [hunk 5]
```diff
@@ -21,6 +22,18 @@
   }
 }
 
+// @media screen and (max-width: 850px) {
```
Comment (kiljee) [inline src/components/nav/Nav.scss]: remove comments

File: src/components/nav/NavLinks.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+import "./Nav.scss";
+
+export default function NavLinks({
+  hide = false,
+  closeMobile = () => {},
+  openLogin,
+  openNewAcc,
+}) {
+  return (
+    <div className={`link-container ${hide && "link-container--hide"}`}>
```
Comment (kiljee) [inline src/components/nav/NavLinks.jsx:13]: nice

File: src/index.scss [hunk 7]
```diff
@@ -21,3 +22,16 @@ body {
 //   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
 //     monospace;
 // }
+
```
Comment (kiljee) [inline src/index.scss]: remove comments

**Comment by iObrovac** [comment]
@kiljee

**Comment by iObrovac** [comment]
@ognjens022

**Pull request merged**